recipes/selene-sdk/meta.yaml

```diff
@@ -35,11 +35,15 @@ requirements:
     - torchvision
 
 test:
-    imports:
-        - selene_sdk
+  imports:
+    - selene_sdk
 
 about:
   home: https://github.com/FunctionLab/selene
   summary: Framework for developing sequence-level deep learning networks.
-  license: BSD 3-clause
+  license: BSD 3-clause clear
```

Comment:
clear?

Reply:
Yes, we've updated our license: https://github.com/FunctionLab/selene/blob/master/LICENSE

Reply:
Ah, no patent rights. Got it.